client/src/components/UserPlaylistDetail/UserPlaylistDetail.jsx

```diff
@@ -166,6 +163,13 @@ function UserPlaylistDetail () {
           playlistName={currentAddPlaylist === null ? playlist.name : currentAddPlaylist.playlist.name}
         />
       }
+      {profilePopup && profile && spotifyProfile &&
```

Comment:
I think it's better to not do an if statement because I don't have an else condition. I also don't have to create a variable to hold the jsx element. 
https://handsonreact.com/docs/conditional-rendering#:~:text=traditional%20templating%20language)-,is%20less%20code%20than%20an%20if%20because%20it%20is%20not%20necessary%20to%20create%20a%20variable%20to%20hold%20the%20JSX%20element,-When%20there%20isn%27t